sonar-php-plugin/src/test/resources/org/sonar/plugins/php/phpunit/sensor/phpunit.systemout.xml

```diff
@@ -0,0 +1,114 @@
+      </testcase>
+    </testsuite>
+    <testsuite name="BoundingboxTest" time="0.000423">
+      <testcase name="testBoundingbox" time="0.000423"/>
+    </testsuite>
+    <testsuite name="featureRequestTest" time="0.001885">
+      <testcase name="testSetFilters" time="0.000413"/>
+      <testcase name="testGetUrl" time="0.000211"/>
+      <testcase name="testGetFilter" time="0.001261"/>
+    </testsuite>
+    <testsuite name="wfsSearchValuesTest" time="0.001239">
+      <testcase name="testAdavancedBis" time="0.000441"/>
+      <testcase name="testAdavancedStations" time="0.000374"/>
+      <testcase name="testSearchKeyValuePair" time="0.000424"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\52RendererTest" time="0.063931">
+      <testcase name="testTimeSeriesDataXMLToJSON" time="0.063931"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\AquaRendererTest" time="0.003571">
+      <testcase name="testTimeSeriesDataXMLToJSON" time="0.003571"/>
+    </testsuite>
+    <testsuite name="OGCRendererTest" time="0.000797">
+      <testcase name="testOWSException" time="0.000797"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\SensorObsRendererTest" time="0.002452">
+      <testcase name="testSetFlavour_Aqua" time="0.001171"/>
+      <testcase name="testSetFlavour_52" time="0.001096"/>
+      <testcase name="testSetFlavour_wrong_renderer" time="0.000185"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\StationRendererTest" time="0.002600">
+      <testcase name="testDetailsToJson" time="0.002600"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\TerminalRemoteTest" time="0.006743">
+      <testcase name="testAutocomplete" time="0.006743"/>
+    </testsuite>
+    <testsuite name="App\MyBundle\Tests\Services\UserServiceTest" time="0.002339">
+      <testcase name="testCheckRegistered_null_user" time="0.000214"/>
+      <testcase name="testCheckRegistered_wrong_object" time="0.000556"/>
+      <testcase name="testCheckRegistered_no_roles" time="0.000623"/>
+      <testcase name="testCheckRegistered_many_roles" time="0.000230"/>
+      <testcase name="testCheckRegistered_pending_roles" time="0.000240"/>
+      <testcase name="testCheckRegistered_active_roles" time="0.000246"/>
+      <testcase name="testCheckRegistered_disabled_roles" time="0.000230"/>
+    </testsuite>
+    <testsuite name="SourcesTest" time="0.000315">
+      <testcase name="testGetAll" time="0.000315"/>
+    </testsuite>
+  </testsuite>
+</testsuites>
```

Comment:
@kinow git likes an empty line at the end of each file (see https://jira.sonarsource.com/browse/RSPEC-113)